mysql-test/main/mysqld--help.result

```diff
@@ -95,6 +95,10 @@ The following specify which files/extra groups are read (specified before remain
  statement-based binary logging (smaller binary logs),
  MIXED for statement-based binary logging when it's safe
  with fall back to row-based otherwise
+ --binlog-free-flush-threshold=# 
```

Comment:
I think `binlog-free-flush-threshold` is a little hard to decipher off the bat. Perhaps if we leverage the existing binlog group commit convention, this can be `binlog_group_commit_threshold`, which defines the size for transactions that, when smaller are considered for group commit , and when larger, commit standalone using their own transaction cache as a new binary log file. Though before making any changes, @andrelkin and @knielsen should provide input too.